pkg/kudoctl/cmd/params/parser.go

```diff
@@ -51,35 +51,42 @@ func getParamsFromFiles(fs afero.Fs, filePaths []string, errs []string) (map[str
 			errs = append(errs, fmt.Sprintf("error reading from parameter file %s: %v", filePath, err))
 			continue
 		}
-		data := make(map[string]interface{})
-		err = yaml.Unmarshal(rawData, &data)
+
+		errs = GetParametersFromFile(filePath, rawData, errs, parameters)
+
+	}
+	return parameters, errs
+}
+
+func GetParametersFromFile(filePath string, bytes []byte, errs []string, parameters map[string]string) []string {
```

Comment:
Split this part out of the `getParamsFromFiles` method because I needed it for `paramsFile` expansion.